src/Microsoft.OData.Core/UriParser/ODataUriParser.cs

```diff
@@ -607,6 +603,26 @@ private void InitQueryOptionDic()
             }
         }
 
+        /// <summary>
+        /// Generate a list of <see cref="CustomQueryOptionNode"/> from the custom query options.
+        /// </summary>
+        /// <param name="customQueryOptions"></param>
+        /// <returns></returns>
+        private static List<CustomQueryOptionNode> CreateCustomQueryOptionNodes(IList<KeyValuePair<string, string>> customQueryOptions)
+        {
+            var customQueryOptionNodes = new List<CustomQueryOptionNode>();

```

Comment:
In ODL, we use explicit type instead of `var` unless it is a test

```suggestion
            List<CustomQueryOptionNode> customQueryOptionNodes = new List<CustomQueryOptionNode>();

```